packages/typespec-ts/test/integration/modelPropertyOptional.spec.ts

```diff
@@ -305,6 +305,349 @@ describe("ModelsPropertyOptional Rest Client", () => {
     }
   });
 
+  it("should get all string literal", async () => {
+    try {
+      const result = await client
+        .path("/type/property/optional/string/literal/all")
+        .get();
+      assert.strictEqual(result.status, "200");
+      assert.strictEqual(result.body.property, "hello");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it("should get default string literal", async () => {
+    try {
+      const result = await client
+        .path("/type/property/optional/string/literal/default")
+        .get();
+      assert.strictEqual(result.status, "200");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it("should put all string literal", async () => {
+    try {
+      const result = await client
+        .path("/type/property/optional/string/literal/all")
+        .put({
+          body: { property: "hello" }
+        });
+      assert.strictEqual(result.status, "204");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it("should put default string literal", async () => {
+    try {
+      const result = await client
+        .path("/type/property/optional/string/literal/default")
+        .put({
+          body: {}
```

Comment:
why empty?

Reply:
The default value is set to empty like the other cases. For more details check here: https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/type/property/optionality/mockapi.ts#L22